lib/Minz/FrontController.php

```diff
@@ -106,9 +106,10 @@ public function run () {
 	*/
 	private function killApp ($txt = '') {
 		if ($txt == '') {
-			$txt = 'See logs files';
+			exit (errorMessage ("Application problem") );
```

Comment:
Here, you do not need the `if` part anymore since you handle everything in the `errorMessage` function. You can now simplify the method by writing only the following:
```php
private function killApp ($txt = '') {
	exit(errorMessage('Application problem', $txt);
}
```